src/components/WebexMediaAccess/WebexMediaAccess.scss

```diff
@@ -1,25 +1,37 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-media-access {
   display: flex;
-  align-items:center;
-  justify-content: center;
   flex-direction: column;
 
-  .title {
-    margin-top: 1.2rem;
-    font-size: .875rem;
-    font-weight: bold;
+  .logo-container {
+    position: absolute;
```

Comment:
is the `.media-access-body` container still needed if this element is absolutely positioned?